packages/cerebral/src/viewFactories/react.js

```diff
@@ -4,7 +4,23 @@ import StateContainerFactory from './StateContainer'
 import HocFactory from './Hoc'
 import connectFactory, { decoratorFactory } from './connect'
 
-export const Container = ContainerFactory(React)
-export const StateContainer = StateContainerFactory(React)
-export const connect = connectFactory(HocFactory(React))
-export const decorator = decoratorFactory(HocFactory(React))
+let PropTypes
+
+if (React.version.slice(0, 4) === '15.5') {
+  try {
+    PropTypes = require('prop-types')
```

Comment:
Maybe 
```js
if ((major === '15' && minor >= 5) || major >= '16')
```
So it doesn't need to be adjusted on every major react release. Only if it breaks.